src/api/api.js

```diff
@@ -170,29 +175,35 @@ class Api {
     });
   }
 
-  async userLogin(data) {
+  async sendHttpPromise(method, endpoint, data) {
```

Comment:
let's do `sendHttpPromise` as private method and have 2 public:
- userLogin
- userLogout